mkdocs.yml

```diff
@@ -1,22 +1,26 @@
 site_name: Viadot Docs
-site_url: "https://dyvenia.github.io/viadot/"
+site_url: "https://djagoda881.github.io/viadot/"
 nav:
   - Home: index.md
   - Howtos:
```

Comment:
[📝 Improved mkdocs.yml](https://github.com/dyvenia/viadot/pull/657/commits/13865948dd012e4323bff985fe6783c7a15f98b0)

Reply:
you forgot to rename the folders

Reply:
[📝 Added migration guide in User Guide](https://github.com/dyvenia/viadot/pull/657/commits/fa7989fc32e55328e16cb2ce727124c05aca525c)

I also added a description of the migration process that I left out. I think it may be helpful for the moment and in the future after the process is completed we can remove it

Reply:
Prefect stuff shouldn't be here -please remove it.

Reply:
[📝 Improved migration_guide](https://github.com/dyvenia/viadot/pull/657/commits/cef9cae144bb01548c5ae28c303274ef95ffc1c9)

It was left prefect in the section explaining that references to it should be removed during migration.  I also removed all references to the prefect-viadot repo